ControlCenterUi/src/Pages/ColorSelectPage.tsx

```diff
@@ -49,6 +54,24 @@ const ColorSelectPage: FC = () => {
     });
   };
 
+  const RainbowRequest: RainbowFormat = {
+    operation: "rainbow",
+    wait_ms: 20
```

Comment:
I wouldn't even include wait_ms; not needed since it's nullable

Reply:
Naming convention for variables is camelCase.

So here, would be named `rainbowRequest`